community/project committers.md

```diff
@@ -43,3 +43,19 @@
   | Weiwei Yang     | Apple        |
   | Zhankun Tang    | Cloudera     |
   | Mao Baolong     | Tencent      |
+
+## how to become a committer
```

Comment:
Beyond describing existing contributors and PPMC members, add a section on how new contributors are attracted, for example:

"To attract new contributors, we regularly host open source contributor meeting, offer a mentorship program, and actively respond to every message on PRs to foster an environment that welcomes and supports newcomers."

Reply:
OK.